client/app/widgets/Comments/__tests__/Comments.spec.jsx

```diff
@@ -106,7 +106,7 @@ describe('Comments', () => {
       await waitFor(() => expect(screen.getByRole('article')).toBeInTheDocument());
       expect(screen.getByRole('article')).toHaveTextContent('Hey');
 
-      userEvent.click(screen.getByRole('link', { name: 'Delete' }));
+      userEvent.click(screen.getByRole('link', { name: `Delete ${value}` }));
```

Comment:
This test also bring up another edge case, we don't want to pass in the entire comment text here. Here, it makes more sense to have something like "Delete from comment from [the user the comment is by]".